.gitignore

```diff
@@ -47,5 +47,16 @@ VERSION
 
 .DS_Store
 .vscode/
+.idea
 .venv
+bindings/python/dist
```

Comment:
Who pushed these things in past that we need them in `.gitignore`? (I think it is good idea, just curious, who did it first :D)

Reply:
No one pushed them to the repo. That is why they keep showing as unstaged changes in the working tree. I would like to prevent this behaviour.

Reply:
Ah, ok. I just wanted to know, what made you pick these files in this PR.

Reply:
(I suspected, certain someone pushed them in past :D)

Reply:
I just pushed some ignore rules I have in my local `.gitignore` which made sense to ignore globally.